src/lib/object-class.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the object class for any value.
+ *
+ * @param {*} [value] - A value whose object class should be determined.
+ *
+ * @example
+ * const isNull = objectClass(null) === 'null'
+ * const isPOJO = objectClass({}) === 'object'
+ *
+ * @returns {String}
+ */
+const objectClass = (value) => /\[\w+ (\w+)]/.exec(Object.prototype.toString.call(value))[1].toLowerCase()
```

Comment:
@levithomason honestly, my position aside, I'd compromise in a heartbeat on something like `getObjectClass`, just because it describes the action it's performing. It's also an example from that SO post you linked. Leaving this to you!